.travis.yml

```diff
@@ -0,0 +1,26 @@
+os:
+  - linux
+  - osx
+
+language: go
+sudo: false
+
+go:
+  - 1.7
+
+before_install:
+  - go get github.com/mattn/goveralls
+  - go get golang.org/x/tools/cmd/cover
+  - go get github.com/whyrusleeping/gx
+  - go get github.com/whyrusleeping/gx-go
```

Comment:
copied this from `go-libp2p-transport`, so should be changed there as well probably